### DISCUSSION TITLE
Teleportation using Unity XR Interation Toolkit

### DISCUSSION THREAD

**Discussion by Ram1508** [MixedRealityToolkit/MixedRealityToolkit-Unity#478] [Q&A]
@marlenaklein-msft 

**Scenario***
I was trying out the Teleportation system provided by unity XR Interaction toolkit as mentioned in  https://learn.unity.com/tutorial/locomotion-and-teleportation-in-the-xr-interaction-toolkit.
I followed the instructions as mentioned above.

1) Added Locomotion object as child to camera offset( which has XR Orgin) and Teleportation provider.
2) Created a plane, added teleportation area component and marked its collider in the section given.
3) Interaction layer is set as Everything.

**Problem**

I wasn't able to teleport at all and receive this message : 
 
![Screenshot (233)](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/131968892/a52ff024-5fb7-4521-8ddd-5e2302c4ab9c)

**Version Used**
Unity Version : 2021.3.16f1

Visual Studio : 2019 (16.11.25)

Is there anything am i missing?

**Comment by marlenaklein-msft** [comment]
Hi Ram,

When does the UnboundedTrackingMode warning appear? Do you still see this if Locomotion System and Teleportation Provider/Area are not in the scene? Also, would be good to know the version of MRTK3 you're using. 

Since the tutorial isn't using MRTK, for questions about the tutorial part I would recommend posting on the [Unity Forum](https://forum.unity.com/forums/ar-vr-xr-discussion.80/).

**Comment by Ram1508** [comment]
@marlenaklein-msft 

Thanks for the reply. 

The issue was with the XR interaction toolkit version. Once I updated it was fixed.

**Comment by abhilashca** [comment]
@Ram1508 Did you implemented teleportation? Is that point and click behavior for teleporting? Where you able to achieve teleportation arc just like in MRTK2?

It would be great if you can take a look at these.
-  https://github.com/orgs/MixedRealityToolkit/discussions/613
- https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/issues/615

Both are discussing the same.

I tried replicating XRI teleportation, but MRTK3 teleportation is working using point-and-click only.
